reverse_strand.py

```diff
@@ -0,0 +1,36 @@
+#! /usr/bin/python3
+import argparse
+from argparse import RawTextHelpFormatter
+import bamnostic as bs
+
+description="Reverses the allele for variants that got mapped onto the reverse strand of the new genome, and prints everything to a new file\nAlso filters out the non-mapped"
```

Comment:
Could you please wrap the line length throughout the code to the maximum of 120 columns?

Reply:
I'm not sure how to set the maximum in Sublime Text 3, however I have set `word_wrap true` which adds auto line wrapping to the files.

Reply:
How to set maximum line length in Sublime: https://stackoverflow.com/questions/25900954/80-characters-right-margin-line-in-sublime-text-3 (use 120 instead of 80 as described there)

It looks like the option you currently have enabled wraps the lines to some other limit (or doesn't at all), bercause for example line 157 of the `remapping_commands.sh` is 145 characters long